docs/patterns/components/Tabs/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Tabs, Tab, TabPanel, Code } from '@deque/cauldron-react';
+import './index.css';
+
+const Demo = () => (
+  <div className="TabsDemo">
+    <h1>Tabs</h1>
+    <h2>Demo</h2>
+    <h3>Basic Tabs</h3>
+    <Tabs>
```

Comment:
Shouldn't the demo not omit required props? I think we should add an `aria-label` or `aria-labelledby` here